study.md

```diff
@@ -23,13 +23,13 @@ Array methods?
 Similarities:
 
 ```md
-<!-- your answer here -->
+<!-- New elements are added to the end of the array. -->
 ```
 
 Differences:
 
 ```md
-<!-- your answer here -->
+<!-- To find index of elements in js indexOf is used; instead, in ruby index. -->
```

Comment:
This has nothing to do with the question being asked.

Ruby's push array method returns the modified array, allowing method chaining.

JavaScript's push array method returns the length of modified array.